### DISCUSSION TITLE
Transfering ownership of autofirma-nix to nix-community

### DISCUSSION THREAD

**Discussion by nilp0inter** [nix-community/infra#1627] [Q&A] [answered]
Hi!

I'm @nilp0inter, author of [autofirma-nix](https://github.com/nilp0inter/autofirma-nix).

autofirma-nix provides NixOS and Home Manager modules for tools required to interact with Spain's public administration services. The project is actively maintained by myself, @panchoh, and @CesarGallego.

Would it be possible to move `autofirma-nix` under the nix-community organization?

**Comment by Mic92** [comment]
Invite sent

**Comment by nilp0inter** [comment, marked as answer]
Thank you for the quick reply, @Mic92 !

Repo transferred: https://github.com/nix-community/autofirma-nix

**Comment by Mic92** [comment]
I setup a team and restored your admin rights. I invited your external contributors as team members as well. After accepting they can write to the repository again. You can also add more people to https://github.com/orgs/nix-community/teams/autofirma-nix

**Comment by nilp0inter** [comment]
Thank you very much!
